package com.exceptionprograme;

public class Demo {

	public static void main(String[] args) {

		try {
			
			int i = 10 / 0;

			System.out.println(i);
			
		} catch (ArithmeticException e) {
			
			System.out.println("divide by zero");

			e.printStackTrace();
		}

	}
}
